plugins/pagerduty/README.md

```diff
@@ -95,6 +95,30 @@ annotations:
   pagerduty.com/integration-key: [INTEGRATION_KEY]
 ```
 
+### The homepage component
+
+You may also add the component to the homepage of Backstage. Edit the `HomePage.tsx` file, import `PagerDutyHomepageCard` and add it to the home page. e.g.
+
+```typescript jsx
+...
+export const homePage = (
+  <Page themeId="home">
+    ...
+    <Content>
+      <CustomHomepageGrid config={defaultConfig}>
+        ...
+        <PagerDutyHomepageCard
+          name="My Service"
+          serviceId="<service id>"
+          integrationKey="<integration key>"
```

Comment:
Feels odd to have these both as props and settings right? Wondering what an integrator would put in here statically. Or is it useful for orgs that have only like one or a few services shared for a lot of things?

Reply:
I suppose by doing this you are giving the option of either?